openquake/hazardlib/tests/geo/polygon_test.py

```diff
@@ -288,6 +288,15 @@ def test_points_close_to_corners(self):
 
         self.assertTrue(self.poly.intersects(mesh).all())
 
+class PolygonFromWktTestCase(unittest.TestCase):
+    def test(self):
+        wkt_string = 'POLYGON((22. -15.,24. -15.,24. -10.,22. -15.))'
+        poly = polygon.Polygon.from_wkt(wkt_string)
+        self.assertEqual(list(poly.lats), [-15, -15, -10])
+        self.assertEqual(list(poly.lons), [22, 24, 24])
+        self.assertEqual(poly.lats.dtype, 'float')
+        self.assertEqual(poly.lons.dtype, 'float')
+        self.assertTrue(poly.wkt != '')
```

Comment:
I would change the last line to

`self.assertEqual(poly.wkt, 'POLYGON((22.0 -15.0, 24.0 -15.0, 24.0 -10.0, 22.0 -15.0))')`